src/Service/UpdateWithParamsTrait.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace ChartMogul\Service;
+
+use ChartMogul\Http\ClientInterface;
+
+/**
+* @codeCoverageIgnore
+*/
+trait UpdateWithParamsTrait
+{
+    public static function update(array $id = [], array $data = [], ClientInterface $client = null)
```

Comment:
Just to avoid confusion I would suggest changing the name (and removing id as a separate argument)
```suggestion
    public static function updateWithParams(array $data = [], ClientInterface $client = null)
```